### DISCUSSION TITLE
Will photoprisms database be the same size as storage?

### DISCUSSION THREAD

**Discussion by Daniel-PT** [photoprism/photoprism#3939] [Q&A]
Hi all. If i have about 1TB pictures/videos in my /originals. Will the database for photoprism be 1 TB ? :)

Thanks!

**Comment by lastzero** [comment]
see https://docs.photoprism.app/getting-started/faq/#why-is-my-storage-folder-so-large-what-is-in-it

**Reply by Daniel-PT** [reply]
Meaby im dumb or something but i cant figure out if 1TB storage will fill 1TB into the database? :)

**Reply by lastzero** [reply]
Certainly not the database, but let's say you have 1TB of HVEC encoded videos and no browser that can play them, so you need to transcode them to AVC. AVC is 30% less efficient, which means you need 1.3 TB of extra disk space. This is the worst case scenario. Note that your originals can be indexed directly, see our docs. So they are not duplicated 1:1, if that was your question:

https://docs.photoprism.app/user-guide/first-steps/

**Reply by Daniel-PT** [reply]
Ah okay meny thanks :)

Was just affraid that the database will increase. But yea that is the diffrence between import and index :)

Thanks!

**Reply by Daniel-PT** [reply]
Can i somehow see if it is transcoding? Can i use like the CPU/mem and not storage so i wont "convert" the video? :)
